addon/v1alpha1/types_clustermanagementaddon.go

```diff
@@ -66,6 +66,11 @@ type ConfigCoordinates struct {
 	// This field should be configured if add-on CR have a consistent name across the all of the ManagedCluster instaces.
 	// +optional
 	CRName string `json:"crName"`
+
+	// lastObservedGeneration is the object generation of the config reference custom resource
```

Comment:
observed generation of the custom resource for the configuration of the addon

Reply:
updated